### PR TITLE
Improvements Installation manual. Updated manual to reflect changes f…

### PR DESCRIPTION
**Pull request by pczeeuw**
…rom OC-143

### CODE REVIEW

**Comment by pczeeuw** [comment]
PR for changes in Config is still pending review.

Most of these changes reflect the changes made in the puppet script (OC-143).
- Updated procedure for adding SoapUi project.
- Made command line execution of certain tools 'optional' (since there are shortcuts now)
- Added system requirement.

File: Userguide/Installation/Setup-VM-Vagrant.md [hunk 1]
```diff
@@ -32,10 +43,15 @@ Navigate to your newly created directory and choose 'save'.
 
 #### Note
  - Make sure that the file is named like this: `Vagrantfile`  *without* an extension!
- - If the file has an extension (for example .txt) you can rename the file using the following console command:
+ - If the file has an extension (for example .txt) you can rename the file using the following console command.
+Unix:
```
Comment (Sander3003) [inline Userguide/Installation/Setup-VM-Vagrant.md]: MacOS/Linux

**Comment by Sander3003** [comment]
Looks good to me!

File: Userguide/Installation/Setup-VM-Vagrant.md (hunk 1 above)
Comment (pczeeuw) [inline Userguide/Installation/Setup-VM-Vagrant.md]: Changed. I will merge this with the documentation as soon as the PR for OC-149 has been approved.

**Pull request merged**